src/QMCTools/ppconvert/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# this new link path policy has fixed ldpath problems
+# experienced with out of dyld_load_path libraries on
+# OSX without bad effect on other platforms.
+cmake_policy(SET CMP0003 NEW)
```

Comment:
Do we really need cmake_policy? What happends if this is removed completely.

Reply:
It's gone for this PR